src/categories/views/CategoryDetails.tsx

```diff
@@ -327,9 +367,9 @@ export const CategoryDetails: React.FC<CategoryDetailsProps> = ({
             id="Pp/7T7"
             defaultMessage="{counter,plural,one{Are you sure you want to delete this category?} other{Are you sure you want to delete {displayQuantity} categories?}}"
             values={{
-              counter: maybe(() => params.ids.length),
+              counter: maybe(() => selectedCategoryRowIds.length),
               displayQuantity: (
-                <strong>{maybe(() => params.ids.length)}</strong>
+                <strong>{maybe(() => selectedCategoryRowIds.length)}</strong>
```

Comment:
**nitpick** if you're touching lines with `maybe` it might be good to replace it with optional chaining / nullish coalescing